docs/docs-content/clusters/edge/site-deployment/site-installation/site-user-data.md

```diff
@@ -98,29 +98,34 @@ to a bootable device, such as a USB stick.
 
       </Tabs>
 
-    This generates an ISO file called **site-user-data.iso** in the current directory.
+    This generates an ISO file called `site-user-data.iso` in the current directory.
 
 4.  Flash your bootable device such as a USB drive with the ISO file you just created.
 
-    :::info
-
     You can use several software tools to create a bootable USB drive, such as
     [balenaEtcher](https://www.balena.io/etcher). For a PXE server, there are open source projects such as
     [Fog](https://fogproject.org/download.php) or
     [Windows Deployment Services](https://learn.microsoft.com/en-us/windows/deployment/wds-boot-support) for Windows.
 
+    :::info
+
+    The site user data ISO file is not bootable. It contains only configuration data, which the system reads after
+    booting from the internal disk. If you use a tool like balenaEtcher to write the ISO file to a USB stick, it may
```

Comment:
```suggestion
    booting from the internal disk. If you use a tool like [balenaEtcher](https://etcher.balena.io/) to write the ISO file to a USB stick, it may
```